meta/recipes-connectivity/dhcp/dhcp_4.4.1.bb

```diff
@@ -8,7 +8,6 @@ SRC_URI += "file://0001-define-macro-_PATH_DHCPD_CONF-and-_PATH_DHCLIENT_CON.pat
             file://0007-Add-configure-argument-to-make-the-libxml2-dependenc.patch \
             file://0009-remove-dhclient-script-bash-dependency.patch \
             file://0012-dhcp-correct-the-intention-for-xml2-lib-search.patch \
-            file://0013-workaround-busybox-limitation-in-linux-dhclient-script.patch \
```

Comment:
Again; why the revert?

Reply:
I think it was already coming in from a later release

Reply:
In sumo we use the dhcp packages to manage networking, I think. Can you confirm that the `.patch` is no longer needed, and give that justification in your commit message?

Reply:
I updated the commit to reflect that we revert it so that the upstream version will apply cleanly.

a311013e1fd0e7b874dfe11bb3cdf4839cd91c9f